it/mongodb/build.gradle

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * License); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+plugins { id 'org.apache.beam.module' }
+applyJavaNature(
+        automaticModuleName: 'org.apache.beam.it.mongodb',
+)
+
+description = "Apache Beam :: IT :: MongoDB"
+ext.summary = "Integration test utilities for MongoDB."
+
+dependencies {
+    implementation project(':it:common')
+    implementation library.java.testcontainers_base
+    implementation library.java.testcontainers_mongodb
+    implementation library.java.vendored_guava_26_0_jre
+    implementation library.java.slf4j_api
+    //implementation 'org.mongodb:mongodb-driver-sync:3.12.10'
+    implementation library.java.mongo_java_driver
+    implementation 'com.google.re2j:re2j:1.6'
```

Comment:
same comment here - why use the Google library? should at least have a good reason and comment

Reply:
Fixed in https://github.com/GoogleCloudPlatform/DataflowTemplates/pull/710.